format/Layout.md

```diff
@@ -113,9 +114,9 @@ j mod 8   7  6  5  4  3  2  1  0
           0  0  1  0  1  0  1  1
 ```
 
-Physically, non-nullable (NN) arrays do not have a bitmap.
+Physically, non-nullable (NN) arrays do not have a null bitmap.
```

Comment:
Totally. I'm going to make another patch to strike nullability from the memory layout, but didn't want to pack too many changes into this patch https://issues.apache.org/jira/browse/ARROW-76